### ISSUE TITLE
Unexpected parse error in begining of file

### ISSUE THREAD

**Issue by rbaprado** [sasstools/sass-lint#969]
I am getting a "Unexpected parse error in file" at the first character of my file:
```
/**
 * Base definitions
 */

* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}
```
Note: I am running sass-lint through the Atom linter plugin.

This seems related to #580

**Comment by DanPurdy** [comment]
Parse errors like this are thrown by the AST we use, [Gonzales-Pe](https://github.com/tonyganch/gonzales-pe)

As far as I can see this is parsing fine in the latest build, therefore we may just have to wait until that version of the AST has stabilised and we can incorporate it into sass-lint.

**Comment by bgriffith** [comment]
Not sure what was causing this issue but it is fixed in the  newly released version v1.11.0. Please let me know if you still have issues. Thanks!
